build/musl-cross/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM debian:jessie
+
+RUN apt-get update && \
+	apt-get install -y git build-essential gawk wget \
+		libmpfr-dev libgmp-dev libmpc-dev
+
+RUN mkdir -p /cross-build/conf && \
+	cd /cross-build && \
+	git clone https://github.com/sabotage-linux/musl-cross.git && \
+	cd musl-cross && git checkout a4777a6
+
+ADD amd64 /cross-build/conf/amd64
```

Comment:
I think it's clearer if you rename `amd64/config.sh` to `config_amd64.sh` and `ADD config_amd64.sh /cross-build/conf/amd64/config.sh`. No extra dirs are created then